.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2020/11 to 2020/12
+x. Enhancement - Update EGWU Northolt SMR - thanks to @scottdaniels292 (Scott Daniels)
```

Comment:
```suggestion
x. Enhancement - Added new Northolt (EGWU) SMR - thanks to @scottdaniels292 (Scott Daniels)
```

The SMR is technically new, because before it was only an old-style ground map :) So give yourself the full credit!